modules/fakemeta/fakemeta_amxx.cpp

```diff
@@ -39,13 +42,27 @@ void OnAmxxAttach()
 
 	ConfigManager = MF_GetConfigManager();
 
-	char error[256];
-	error[0] = '\0';
+	char error[256] = "";
 
 	if (!ConfigManager->LoadGameConfigFile("common.games", &CommonConfig, error, sizeof(error)) && error[0] != '\0')
 	{
 		MF_Log("Could not read common.games gamedata: %s", error);
-		MF_Log("get/set/find_ent_data* natives have been disabled");
+		MF_Log("get/set_ent_data* natives have been disabled");
```

Comment:
On purpose because `MF_Log` includes automatically EOL and that a line in the log the date and time is automatically inserted before, so if you add manually EOL here it kind of breaks visually the log.